lib/bas/shared_storage/postgres.rb

```diff
@@ -26,11 +26,16 @@ def read
       end
 
       def write(data)
-        params = { connection: write_options[:connection], query: write_query(data) }
+        return if write_options[:avoid_empty_data] && empty_data?(data)
```

Comment:
I'm not sure if these conditions should be part of the bot or the shared storage. I think this condition should be a bot configuration since is something you want to do at a bot level. Also, if we add a new shared storage then we will have to add conditions like this. What do you think? :thinking: